ramls/accounts.raml

```diff
@@ -1,6 +1,6 @@
 #%RAML 1.0
 title: Accounts
-version: v1
+version: v1.1
```

Comment:
All the RAML in this module have the same version since their inception - `v1`, which was never updated by previous module maintainers. Also, all the endpoings described in these RAMLs are exposed through one inteface - `feesfines`. So incremented version of the RAML will not be equal to the version of the interface in `ModuleDescriptor`. In these circumstances, do you think starting updating RAML versions now is the right thing to do?

@marcjohnson-kint I'd love to hear your thoughts too.

Reply:
I think yes, I'd update the RAML with 16.1 then, just to match version in the descriptor file.

Reply:
But then should we update versions of all RAMLs to `v16.1` as well?

Reply:
@OleksandrVidinieiev 

> All the RAMLs in this module have had the same version since their inception - v1, which was never updated by previous module maintainers.

This is a fairly common occurrence unfortunately, folks (including myself) often forget to change the RAML version.

> Also, all the endpoints described in these RAMLs are exposed through one interface - feesfines

My understanding is that there should be only a single RAML file for each FOLIO interface.

I think to resolve this discrepancy either
* the existing RAML files should be consolidated into a single file with version `v16.1`
* change the descriptor so it provides an interface for each of the RAML files

The latter of these would be a breaking compatibility change. The former should hopefully not change the behaviour (however I don't know that for sure, due to the way FOLIO uses code generation from RAML files).

If we don't want to do either of these at the moment, then let's at least make the RAML files reflect the single version that we currently expose in the system.

(This is likely an unfortunate side effect of how the system does use the RAML files at all, interfaces only exist in the form of entries in a descriptor)

Reply:
>    the existing RAML files should be consolidated into a single file with version v16.1
>    change the descriptor so it provides an interface for each of the RAML files

Either one of these is out of scope of the story Bogdan is working on. I think I will create a tech-debt ticket to deal with this issue properly.

> If we don't want to do either of these at the moment, then let's at least make the RAML files reflect the single version that we currently expose in the system.

Yes, let's do that.

Reply:
I've update the version in `accounts.raml`. I assume the tech-debt will cover rest of the files?

Reply:
Yes, thanks.